raster/src/main/scala/geotrellis/raster/ArrayMultibandTile.scala

```diff
@@ -428,4 +428,16 @@ class ArrayMultibandTile(_bands: Array[Tile]) extends MultibandTile with MacroMu
 
     new ArrayMultibandTile(newBands)
   }
+
+  override def equals(other: Any): Boolean = other match {
+    case that : ArrayMultibandTile =>
+      var result = (bandCount == that.bandCount)
```

Comment:
I've checked the null case in the console and if I'm not mistaken, the null check seems unnecessary because null goes to "case _" part.